examples/tutorials/advanced/focal_mechanisms.py

```diff
@@ -0,0 +1,336 @@
+"""
+Plotting focal mechanisms
+=========================
+
+Focal mechanisms can be plotted as beachballs with the :meth:`pygmt.Figure.meca`
+method.
+
+The focal mechanism data or parameters can be provided as different input types:
+external file, 1-D or 2-D ``numpy.array``, dictionary, ``pandas.Dataframe``.
+Different conventions to define the focal mechanism are supported: Aki and Richards
+(``"aki"``), global CMT (``"gcmt"``), moment tensor (``"mt"``), partial focal
+mechanism (``"partial"``), principal axis (``"principal_axis"``). Please refer to
+the documentation of :meth:`pygmt.Figure.meca` regarding how to set up the input
+data in respect to the chosen input type and convention (i.e the expected column
+order, keys, or column names).
+
+This tutorial focus on how to adjust the display of the beachballs:
+
+- Filling the quadrants
+- Plotting the components of a seismic moment tensor
+- Adjusting the outlines
+- Highlighting the nodal planes
+- Adding offset from the event location
+- Adding a label
+- Using size-coding and color-coding
+"""
+
+# %%
+import pandas as pd
+import pygmt
+
+# Set up arguments for basemap
+size = 5
```

Comment:
Instead of defining `size` and using `region=[-size, size] * 2`, maybe just define `region=[-5, 5, -5, 5]`?

Reply:
You are right, this is easier; updated in commit https://github.com/GenericMappingTools/pygmt/pull/2550/commits/34c392c74552451e2c96a43cb1641c36aaed9b7d.